spec/create-default-events.spec.js

```diff
@@ -0,0 +1,90 @@
+import $ from 'jquery'
+import rangy from 'rangy'
+
+import Cursor from '../src/cursor'
+import Editable from '../src/core'
+
+describe('Default Events', () => {
+  let $elem, editable, focus, blur
+
+  // create a Cursor object and set the selection to it
+  function createCursor (range) {
+    const cursor = new Cursor($elem[0], range)
+    cursor.setSelection()
+    return cursor
+  }
+
+  function createRangeAtEnd (node) {
+    const range = rangy.createRange()
+    range.selectNodeContents(node)
+    range.collapse(false)
+    return range
+  }
+
+  let onListener
+  // register one listener per test
+  function on (eventName, func) {
+    // off() // make sure the last listener is unregistered
+    const obj = { calls: 0 }
+    function proxy () {
+      obj.calls += 1
+      func.apply(this, arguments)
+    }
+    onListener = { event: eventName, listener: proxy }
+    editable.on(eventName, proxy)
+    return obj
+  }
+
+  // unregister the event listener registered with 'on'
+  function off () {
+    if (onListener) {
+      editable.unload()
+      onListener = undefined
+    }
+  }
+
+  describe('for editable', () => {
+    beforeEach(() => {
+      $elem = $('<div contenteditable="true"></div>')
```

Comment:
The `contenteditable` attribute will be added by `editable.add` you can omit it here.